### DISCUSSION TITLE
Welcome to dnxfirewall Discussions!

### DISCUSSION THREAD

**Discussion by DOWRIGHTTV** [DOWRIGHTTV/dnxfirewall#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by x81kilo** [comment]
Any plans to roll out to ARM based devices, e.g. Raspberry Pi or similar SBC platforms?
